### ISSUE TITLE
Add new algorithms.

### ISSUE THREAD

**Issue by D3v3sh5ingh** [D3v3sh5ingh/Graph-Theory-Algorithms#9]

**Comment by Lucifer7355** [comment]
I would like to work on this issue.

**Comment by apurvi96** [comment]
I would Like to add topological sort implementation

**Comment by D3v3sh5ingh** [comment]
Great! Assigning you both.

**Comment by apurvi96** [comment]
@D3v3sh5ingh added PR #10

**Comment by pawki07** [comment]
Hey, I would like to add Kruskal's algorithm in C++. Please assign it to me.

**Comment by D3v3sh5ingh** [comment]
@pawki07 Hey I appreciate your contribution but Krushkal's is already available in to repo.
You can try taking up some other algo, like for example cloning a graph or #7

**Comment by MilindGupta-Creator** [comment]
i would like to add euclid algorithm

**Comment by D3v3sh5ingh** [comment]
@MilindGupta-Creator Great!

**Comment by khushbu1998** [comment]
i would like to add BFS

**Comment by Renu-code** [comment]
I would like to add Djikstra algorithm....can I work on this??

**Comment by D3v3sh5ingh** [comment]
@khushbu1998 @Renu-code Yes, Make sure with code you edit readme file too, so that table of content is also updated simultaneously.

**Comment by Renu-code** [comment]
@D3v3sh5ingh added the djikstra algorithm to the repo and also updated the readme #20  ..please review it

**Comment by princekalra** [comment]
i want to work  on this please assign it to me

**Comment by sumitmohite0501** [comment]
I would like to add minimum spanning tree . please assign me under hacktoberfest label

**Comment by D3v3sh5ingh** [comment]
@sumitmohite0501 You can add and create a pull request, it will be marked hacktoberfest-accepted, if acceptable.
